javascript/javascript1/week3/homework-js1-w3.js

```diff
@@ -114,7 +114,89 @@ logOutSeriesText();
 // NOnoN0nOYes (Note taking app)
 //---------------------------------
 
+// ++++ Save a note ++++
+const notes = [];
+
+function saveNote(content, id) {
+  notes.push({ content: content, id: id });
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
+
+// ++++ Get a note ++++
+function getNote(id) {
+  for (let note of notes) {
+    if (id === note.id) {
+      return note;
+    }
+  }
+}
```

Comment:
Missing the ` If no id is specified or if the id is not a number, log out an error string.` part here.